src/ucp/proto/proto_select.c

```diff
@@ -747,8 +687,9 @@ void ucp_proto_select_short_init(ucp_worker_h worker,
     ssize_t max_short_signed;
     const uint32_t *op_attribute;
 
+    /* Initialize to disabled state to make coverity happy */
+    ucp_proto_select_short_disable(proto_short);
```

Comment:
so can we remove ucp_proto_select_short_disable from the end of this function?

Reply:
AFAIU no, because of the for loop defined later since one of iterations in that for loop can overwrite `proto_short` data and next one can call `out_disable` so we would need to overwrite this field to disable again.